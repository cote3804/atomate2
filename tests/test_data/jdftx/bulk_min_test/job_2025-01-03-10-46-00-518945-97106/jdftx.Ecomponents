   Eewald =     -175.0227652869602935
       EH =       43.7895386148398984
     Eloc =     -136.7089284673805594
      Enl =       10.5303937757487063
     EvdW =       -0.0524097626316296
      Exc =      -64.8649067285547289
 Exc_core =       29.1043301587218401
       KE =       76.0223543716198975
-------------------------------------
     Etot =     -217.2023933245968692
       TS =        0.0005558534505231
-------------------------------------
        F =     -217.2029491780473904
