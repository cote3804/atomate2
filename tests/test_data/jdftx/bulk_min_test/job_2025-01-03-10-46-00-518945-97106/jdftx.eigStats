eMin: -1.341990 at state 9 ( [ +0.500000 +0.500000 +0.000000 ] spin 1 )
HOMO: +0.500298 at state 13 ( [ +0.000000 +0.000000 +0.333333 ] spin -1 )
mu  : +0.513131
LUMO: +0.514670 at state 21 ( [ +0.500000 +0.500000 +0.000000 ] spin -1 )
eMax: +0.922089 at state 14 ( [ +0.000000 +0.000000 -0.333333 ] spin -1 )
HOMO-LUMO gap: +0.014372
Optical gap  : +0.015339 at state 14 ( [ +0.000000 +0.000000 -0.333333 ] spin -1 )
