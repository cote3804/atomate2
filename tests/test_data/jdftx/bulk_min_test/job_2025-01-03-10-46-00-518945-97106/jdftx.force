# Forces in Cartesian coordinates:
force Ir   0.000004781234670   0.000005264959947   0.000000825114056 1
force Ir  -0.000004395831068  -0.000005875483272  -0.000000157164922 1
force O   0.003181520838422   0.003183396098440   0.000000975641323 1
force O   0.003173219018666  -0.003173969497423  -0.000000572004379 1
force O  -0.003182185136984  -0.003182888496619   0.000001647949298 1
force O  -0.003174368517634   0.003174492441906  -0.000000754744825 1

