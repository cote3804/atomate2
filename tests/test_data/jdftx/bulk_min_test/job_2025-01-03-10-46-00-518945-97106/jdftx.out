
*************** JDFTx 1.7.0 (git hash c3f005b2) ***************

Start date and time: Fri Jan  3 02:46:07 2025
Executable /global/cfs/cdirs/m4025/Software/Perlmutter/JDFTx/build-gpu/jdftx_gpu with command-line: -i init.in -o jdftx.out
Running on hosts (process indices):  nid200332 (0)
Divided in process groups (process indices):  0 (0)
gpuInit: Found compatible cuda device 0 'NVIDIA A100-SXM4-80GB'
gpuInit: Selected device 0
Resource initialization completed at t[s]:      0.48
Run totals: 1 processes, 64 threads, 1 GPUs


Input parsed successfully to the following command list (including defaults):

band-projection-params yes no
basis kpoint-dependent
converge-empty-states yes
coords-type Cartesian
core-overlap-check none
coulomb-interaction Periodic
davidson-band-ratio 1.1
dump End State Forces ElecDensity KEdensity Dtot VfluidTot BandEigs BandProjections EigStats Ecomponents BoundCharge DOS
dump-name jdftx.$VAR
elec-cutoff 20 100
elec-eigen-algo Davidson
elec-ex-corr gga-PBE
elec-n-bands 37
elec-smearing Fermi 0.001
electronic-minimize  \
	dirUpdateScheme      FletcherReeves \
	linminMethod         DirUpdateRecommended \
	nIterations          100 \
	history              15 \
	knormThreshold       0 \
	maxThreshold         no \
	energyDiffThreshold  1e-07 \
	nEnergyDiff          2 \
	convergeAll          no \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	abortOnFailedStep    no \
	fdTest               no
exchange-regularization WignerSeitzTruncated
fluid None
fluid-ex-corr lda-TF lda-PZ
fluid-gummel-loop 10 1.000000e-05
fluid-minimize  \
	dirUpdateScheme      PolakRibiere \
	linminMethod         DirUpdateRecommended \
	nIterations          100 \
	history              15 \
	knormThreshold       0 \
	maxThreshold         no \
	energyDiffThreshold  0 \
	nEnergyDiff          2 \
	convergeAll          no \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	abortOnFailedStep    no \
	fdTest               no
fluid-solvent H2O 55.338 ScalarEOS \
	epsBulk 78.4 \
	pMol 0.92466 \
	epsInf 1.77 \
	Pvap 1.06736e-10 \
	sigmaBulk 4.62e-05 \
	Rvdw 2.61727 \
	Res 1.42 \
	tauNuc 343133 \
	poleEl 15 7 1
forces-output-coords Positions
initial-magnetic-moments  \
	Ir 5 5 \
	O 0 0 0 0
ion Ir   4.264424641093000   4.264424641093000   3.003645584499000 1
ion Ir   0.000000083481000   0.000000083481000  -0.000000250174000 1
ion O   5.896306617340000   5.896306617340000  -0.000000250174000 1
ion O   1.631882059728000   6.896967222458000   3.003645584499000 1
ion O   2.632542664846000   2.632542664846000  -0.000000250174000 1
ion O   6.896967222458000   1.631882059728000   3.003645584499000 1
ion-species GBRV_v1.5/$ID_pbe_v1.uspp
ion-width 0
ionic-minimize  \
	dirUpdateScheme      L-BFGS \
	linminMethod         DirUpdateRecommended \
	nIterations          0 \
	history              15 \
	knormThreshold       0.0001 \
	maxThreshold         no \
	energyDiffThreshold  1e-06 \
	nEnergyDiff          2 \
	convergeAll          no \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	abortOnFailedStep    no \
	fdTest               no
kpoint   0.000000000000   0.000000000000   0.000000000000  1.00000000000000
kpoint-folding 2 2 3 
latt-move-scale 0 0 0
latt-scale 1 1 1 
lattice  \
	   8.528849994533999    0.000000000000000    0.000000000000000  \
	   0.000000000000000    8.528849994533999    0.000000000000000  \
	   0.000000000000000    0.000000000000000    6.007289996150000 
lattice-minimize  \
	dirUpdateScheme      L-BFGS \
	linminMethod         DirUpdateRecommended \
	nIterations          0 \
	history              15 \
	knormThreshold       0 \
	maxThreshold         no \
	energyDiffThreshold  1e-06 \
	nEnergyDiff          2 \
	convergeAll          no \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	abortOnFailedStep    no \
	fdTest               no
lcao-params -1 1e-06 0.001
pcm-variant GLSSA13
perturb-minimize  \
	nIterations            0 \
	algorithm              MINRES \
	residualTol            0.0001 \
	residualDiffThreshold  0.0001 \
	CGBypass               no \
	recomputeResidual      no
spintype z-spin
subspace-rotation-factor 1 yes
symmetries none
symmetry-threshold 0.0001
van-der-waals D3


Applied RMS atom displacement 0 bohrs to make symmetries exact.

---------- Initializing the Grid ----------
R = 
[      8.52885            0            0  ]
[            0      8.52885            0  ]
[            0            0      6.00729  ]
unit cell volume = 436.978
G =
[   0.736698          0          0  ]
[          0   0.736698          0  ]
[          0          0    1.04593  ]
Minimum fftbox size, Smin = [  40  40  28  ]
Chosen fftbox size, S = [  40  40  28  ]

---------- Initializing tighter grid for wavefunction operations ----------
R = 
[      8.52885            0            0  ]
[            0      8.52885            0  ]
[            0            0      6.00729  ]
unit cell volume = 436.978
G =
[   0.736698          0          0  ]
[          0   0.736698          0  ]
[          0          0    1.04593  ]
Minimum fftbox size, Smin = [  36  36  28  ]
Chosen fftbox size, S = [  36  36  28  ]

---------- Exchange Correlation functional ----------
Initalized PBE GGA exchange.
Initalized PBE GGA correlation.

---------- Setting up pseudopotentials ----------
Width of ionic core gaussian charges (only for fluid interactions / plotting) set to 0

Reading pseudopotential file '/global/cfs/cdirs/m4025/Software/Perlmutter/JDFTx/build-gpu/pseudopotentials/GBRV_v1.5/ir_pbe_v1.uspp':
  Title: Ir.  Created by USPP 7.3.6 on 3-2-2014
  Reference state energy: -90.436894.  15 valence electrons in orbitals:
    |510>  occupation: 6  eigenvalue: -2.019590
    |520>  occupation: 8.5  eigenvalue: -0.306422
    |600>  occupation: 0  eigenvalue: -0.312524
    |610>  occupation: 0  eigenvalue: -0.124096
  lMax: 2  lLocal: 3  QijEcut: 5
  7 projectors sampled on a log grid with 745 points:
    l: 0  eig: -0.312523  rCut: 2.55
    l: 0  eig: 0.250000  rCut: 2.55
    l: 1  eig: -2.019590  rCut: 1.62
    l: 1  eig: -0.124096  rCut: 1.62
    l: 1  eig: 1.000000  rCut: 1.62
    l: 2  eig: -0.500000  rCut: 1.65
    l: 2  eig: -0.306421  rCut: 1.65
  Partial core density with radius 1.15
  Transforming core density to a uniform radial grid of dG=0.02 with 1279 points.
  Transforming local potential to a uniform radial grid of dG=0.02 with 1279 points.
  Transforming nonlocal projectors to a uniform radial grid of dG=0.02 with 432 points.
  Transforming density augmentations to a uniform radial grid of dG=0.02 with 1279 points.
  Transforming atomic orbitals to a uniform radial grid of dG=0.02 with 432 points.
  Core radius for overlap checks: 2.55 bohrs.

Reading pseudopotential file '/global/cfs/cdirs/m4025/Software/Perlmutter/JDFTx/build-gpu/pseudopotentials/GBRV_v1.5/o_pbe_v1.uspp':
  Title: O.  Created by USPP 7.3.6 on 3-2-2014
  Reference state energy: -15.894388.  6 valence electrons in orbitals:
    |200>  occupation: 2  eigenvalue: -0.878823
    |210>  occupation: 4  eigenvalue: -0.332131
  lMax: 2  lLocal: 2  QijEcut: 6
  5 projectors sampled on a log grid with 511 points:
    l: 0  eig: -0.878823  rCut: 1.25
    l: 0  eig: 0.000000  rCut: 1.25
    l: 1  eig: -0.332132  rCut: 1.25
    l: 1  eig: 0.000000  rCut: 1.25
    l: 2  eig: 1.000000  rCut: 1.25
  Partial core density with radius 0.7
  Transforming core density to a uniform radial grid of dG=0.02 with 1279 points.
  Transforming local potential to a uniform radial grid of dG=0.02 with 1279 points.
  Transforming nonlocal projectors to a uniform radial grid of dG=0.02 with 432 points.
  Transforming density augmentations to a uniform radial grid of dG=0.02 with 1279 points.
  Transforming atomic orbitals to a uniform radial grid of dG=0.02 with 432 points.
  Core radius for overlap checks: 1.25 bohrs.

Initialized 2 species with 6 total atoms.

Folded 1 k-points by 2x2x3 to 12 k-points.

---------- Setting up k-points, bands, fillings ----------
No reducable k-points. 
Computing the number of bands and number of electrons
Calculating initial fillings.
nElectrons:  54.000000   nBands: 37   nStates: 24

----- Setting up reduced wavefunction bases (one per k-point) -----
average nbasis = 1868.750 , ideal nbasis = 1866.803

----- Initializing Supercell corresponding to k-point mesh -----
Lattice vector linear combinations in supercell:
[   2   0   0  ]
[   0   2   0  ]
[   0   0   3  ]
Supercell lattice vectors:
[  17.0577  0  0  ]
[  0  17.0577  0  ]
[  0  0  18.0219  ]

Initializing DFT-D3 calculator:
	Parameters set for gga-PBE functional
	s6:  1.000  s_r6:  1.217
	s8:  0.722  s_r8:  1.000
	Per-atom parameters loaded for:
	Ir:  sqrtQ[a0]:  6.055  Rcov[a0]:  2.098  CN: [ 0.00 1.86 2.94 ]
	 O:  sqrtQ[a0]:  2.594  Rcov[a0]:  1.191  CN: [ 0.00 0.99 1.99 ]

---------- Setting up ewald sum ----------
Optimum gaussian width for ewald sums = 2.245802 bohr.
Real space sum over 891 unit cells with max indices [  4  4  5  ]
Reciprocal space sum over 2925 terms with max indices [  7  7  6  ]

Computing DFT-D3 correction:
# coordination-number Ir 6.673 6.673
# coordination-number O 2.848 2.848 2.848 2.848
# diagonal-C6 Ir 305.53 305.53
# diagonal-C6 O 10.37 10.37 10.37 10.37
EvdW_6 =   -0.021155
EvdW_8 =   -0.031255

---------- Allocating electronic variables ----------
Initializing wave functions:  linear combination of atomic orbitals
Ir (M=5) pseudo-atom majority spin occupations:   s ( 1 )  p ( 3 1 )  d ( 5 )
Ir (M=5) pseudo-atom minority spin occupations:   s ( 1 )  p ( 3 0 )  d ( 1 )
O pseudo-atom occupations:   s ( 2 )  p ( 4 )
	FillingsUpdate:  mu: +0.574499180  nElectrons: 54.000000  magneticMoment: [ Abs: 1.62493  Tot: +0.00000 ]
LCAOMinimize: Iter:   0  F: -216.7558452867719438  |grad|_K:  6.985e-03  alpha:  1.000e+00
	FillingsUpdate:  mu: +0.535105442  nElectrons: 54.000000  magneticMoment: [ Abs: 1.07082  Tot: -0.00000 ]
LCAOMinimize: Iter:   1  F: -216.9840326203547249  |grad|_K:  2.704e-03  alpha:  3.571e-01  linmin:  3.418e-01  cgtest: -8.004e-01  t[s]:      4.18
	FillingsUpdate:  mu: +0.528390384  nElectrons: 54.000000  magneticMoment: [ Abs: 0.69512  Tot: +0.00000 ]
LCAOMinimize: Iter:   2  F: -216.9903440451753340  |grad|_K:  2.811e-03  alpha:  2.867e-01  linmin: -6.731e-04  cgtest:  9.738e-01  t[s]:      4.84
	FillingsUpdate:  mu: +0.514211340  nElectrons: 54.000000  magneticMoment: [ Abs: 0.55330  Tot: -0.00000 ]
LCAOMinimize: Iter:   3  F: -217.0358165758190694  |grad|_K:  3.138e-04  alpha:  2.851e-01  linmin:  9.756e-02  cgtest: -2.302e-01  t[s]:      5.45
	FillingsUpdate:  mu: +0.510990015  nElectrons: 54.000000  magneticMoment: [ Abs: 0.19629  Tot: -0.00000 ]
LCAOMinimize: Iter:   4  F: -217.0374014772173155  |grad|_K:  3.360e-04  alpha:  8.500e-01  linmin: -2.586e-03  cgtest:  9.723e-02  t[s]:      6.13
	FillingsUpdate:  mu: +0.511109033  nElectrons: 54.000000  magneticMoment: [ Abs: 0.07606  Tot: -0.00000 ]
LCAOMinimize: Iter:   5  F: -217.0378309552255018  |grad|_K:  1.182e-04  alpha:  2.193e-01  linmin:  5.088e-02  cgtest: -5.101e-01  t[s]:      6.85
	FillingsUpdate:  mu: +0.511611419  nElectrons: 54.000000  magneticMoment: [ Abs: 0.04015  Tot: -0.00000 ]
LCAOMinimize: Iter:   6  F: -217.0378760177126196  |grad|_K:  9.182e-05  alpha:  2.403e-01  linmin:  3.302e-04  cgtest:  5.814e-01  t[s]:      7.47
	FillingsUpdate:  mu: +0.511829897  nElectrons: 54.000000  magneticMoment: [ Abs: 0.03725  Tot: -0.00000 ]
LCAOMinimize: Iter:   7  F: -217.0379150037922500  |grad|_K:  5.398e-05  alpha:  2.292e-01  linmin:  1.394e-03  cgtest: -2.234e-02  t[s]:      8.11
	FillingsUpdate:  mu: +0.512228144  nElectrons: 54.000000  magneticMoment: [ Abs: 0.03815  Tot: +0.00000 ]
LCAOMinimize: Iter:   8  F: -217.0379389217968935  |grad|_K:  2.895e-05  alpha:  4.089e-01  linmin:  4.289e-04  cgtest:  3.714e-03  t[s]:      8.77
	FillingsUpdate:  mu: +0.512349205  nElectrons: 54.000000  magneticMoment: [ Abs: 0.03880  Tot: +0.00000 ]
LCAOMinimize: Iter:   9  F: -217.0379420605218570  |grad|_K:  1.202e-05  alpha:  1.859e-01  linmin: -1.852e-04  cgtest:  6.605e-03  t[s]:      9.45
	FillingsUpdate:  mu: +0.512380379  nElectrons: 54.000000  magneticMoment: [ Abs: 0.03978  Tot: -0.00000 ]
LCAOMinimize: Iter:  10  F: -217.0379427515036070  |grad|_K:  4.558e-06  alpha:  2.370e-01  linmin: -2.857e-04  cgtest:  6.686e-04  t[s]:     10.08
	FillingsUpdate:  mu: +0.512384754  nElectrons: 54.000000  magneticMoment: [ Abs: 0.04111  Tot: +0.00000 ]
LCAOMinimize: Iter:  11  F: -217.0379429078778628  |grad|_K:  4.411e-06  alpha:  3.730e-01  linmin:  9.735e-05  cgtest:  2.907e-04  t[s]:     10.80
LCAOMinimize: Converged (|Delta F|<1.000000e-06 for 2 iters).


---- Citations for features of the code used in this run ----

   Software package:
      R. Sundararaman, K. Letchworth-Weaver, K.A. Schwarz, D. Gunceler, Y. Ozhabes and T.A. Arias, 'JDFTx: software for joint density-functional theory', SoftwareX 6, 278 (2017)

   gga-PBE exchange-correlation functional:
      J.P. Perdew, K. Burke and M. Ernzerhof, Phys. Rev. Lett. 77, 3865 (1996)

   Pseudopotentials:
      KF Garrity, JW Bennett, KM Rabe and D Vanderbilt, Comput. Mater. Sci. 81, 446 (2014)

   DFT-D3 dispersion correction:
      S. Grimme, J. Antony, S. Ehrlich and H. Krieg, J. Chem. Phys. 132, 154104 (2010)

   Total energy minimization with Auxiliary Hamiltonian:
      C. Freysoldt, S. Boeck, and J. Neugebauer, Phys. Rev. B 79, 241103(R) (2009)

   Linear-tetrahedron sampling for density of states:
      G. Lehmann and M. Taut, Phys. status solidi (b) 54, 469 (1972)

   Smooth electrostatic potentials by atom-potential subtraction:
      R. Sundararaman and Y. Ping, J. Chem. Phys. 146, 104109 (2017)

This list may not be complete. Please suggest additional citations or
report any other bugs at https://github.com/shankar1729/jdftx/issues

Initialization completed successfully at t[s]:     10.93


Computing DFT-D3 correction:
# coordination-number Ir 6.673 6.673
# coordination-number O 2.848 2.848 2.848 2.848
# diagonal-C6 Ir 305.53 305.53
# diagonal-C6 O 10.37 10.37 10.37 10.37
EvdW_6 =   -0.021155
EvdW_8 =   -0.031255

-------- Electronic minimization -----------
	FillingsUpdate:  mu: +0.512384755  nElectrons: 54.000000  magneticMoment: [ Abs: 0.04111  Tot: +0.00000 ]
ElecMinimize: Iter:   0  F: -217.037942907877863  |grad|_K:  3.973e-04  alpha:  1.000e+00
	FillingsUpdate:  mu: +0.519433098  nElectrons: 54.000000  magneticMoment: [ Abs: 0.03845  Tot: +0.00000 ]
	SubspaceRotationAdjust: set factor to 0.999
ElecMinimize: Iter:   1  F: -217.171477602601783  |grad|_K:  2.082e-04  alpha:  5.029e-01  linmin: -1.977e-05  t[s]:     13.32
	FillingsUpdate:  mu: +0.514977579  nElectrons: 54.000000  magneticMoment: [ Abs: 0.03407  Tot: -0.00000 ]
	SubspaceRotationAdjust: set factor to 0.753
ElecMinimize: Iter:   2  F: -217.194714087958545  |grad|_K:  9.071e-05  alpha:  3.224e-01  linmin:  1.198e-04  t[s]:     14.28
	FillingsUpdate:  mu: +0.513089910  nElectrons: 54.000000  magneticMoment: [ Abs: 0.02657  Tot: +0.00000 ]
	SubspaceRotationAdjust: set factor to 0.863
ElecMinimize: Iter:   3  F: -217.201163155035601  |grad|_K:  4.168e-05  alpha:  4.699e-01  linmin:  4.235e-04  t[s]:     15.37
	FillingsUpdate:  mu: +0.512352198  nElectrons: 54.000000  magneticMoment: [ Abs: 0.02242  Tot: +0.00000 ]
	SubspaceRotationAdjust: set factor to 0.869
ElecMinimize: Iter:   4  F: -217.202348419777621  |grad|_K:  2.516e-05  alpha:  4.107e-01  linmin:  1.036e-04  t[s]:     16.42
	FillingsUpdate:  mu: +0.512659961  nElectrons: 54.000000  magneticMoment: [ Abs: 0.01927  Tot: +0.00000 ]
	SubspaceRotationAdjust: set factor to 0.637
ElecMinimize: Iter:   5  F: -217.202687093597490  |grad|_K:  2.154e-05  alpha:  3.206e-01  linmin: -5.563e-05  t[s]:     17.41
	FillingsUpdate:  mu: +0.513431784  nElectrons: 54.000000  magneticMoment: [ Abs: 0.01673  Tot: +0.00000 ]
	SubspaceRotationAdjust: set factor to 0.415
ElecMinimize: Iter:   6  F: -217.202843737059197  |grad|_K:  1.011e-05  alpha:  2.022e-01  linmin:  9.917e-05  t[s]:     18.51
	FillingsUpdate:  mu: +0.513096171  nElectrons: 54.000000  magneticMoment: [ Abs: 0.01385  Tot: -0.00000 ]
	SubspaceRotationAdjust: set factor to 0.497
ElecMinimize: Iter:   7  F: -217.202919596011952  |grad|_K:  5.227e-06  alpha:  4.449e-01  linmin:  3.155e-04  t[s]:     19.62
	FillingsUpdate:  mu: +0.513127672  nElectrons: 54.000000  magneticMoment: [ Abs: 0.01227  Tot: +0.00000 ]
	SubspaceRotationAdjust: set factor to 0.406
ElecMinimize: Iter:   8  F: -217.202938434672660  |grad|_K:  3.203e-06  alpha:  4.141e-01  linmin:  1.132e-06  t[s]:     20.58
	FillingsUpdate:  mu: +0.513161619  nElectrons: 54.000000  magneticMoment: [ Abs: 0.01108  Tot: +0.00000 ]
	SubspaceRotationAdjust: set factor to 0.309
ElecMinimize: Iter:   9  F: -217.202944219670201  |grad|_K:  1.930e-06  alpha:  3.366e-01  linmin: -8.440e-06  t[s]:     21.70
	FillingsUpdate:  mu: +0.513131282  nElectrons: 54.000000  magneticMoment: [ Abs: 0.00974  Tot: -0.00000 ]
	SubspaceRotationAdjust: set factor to 0.366
ElecMinimize: Iter:  10  F: -217.202947176051794  |grad|_K:  1.153e-06  alpha:  4.735e-01  linmin:  3.079e-05  t[s]:     22.75
	FillingsUpdate:  mu: +0.513129029  nElectrons: 54.000000  magneticMoment: [ Abs: 0.00877  Tot: +0.00000 ]
	SubspaceRotationAdjust: set factor to 0.386
ElecMinimize: Iter:  11  F: -217.202948222008217  |grad|_K:  7.343e-07  alpha:  4.704e-01  linmin:  1.177e-05  t[s]:     23.82
	FillingsUpdate:  mu: +0.513127679  nElectrons: 54.000000  magneticMoment: [ Abs: 0.00780  Tot: -0.00000 ]
	SubspaceRotationAdjust: set factor to 0.391
ElecMinimize: Iter:  12  F: -217.202948677171833  |grad|_K:  5.569e-07  alpha:  5.040e-01  linmin:  4.305e-06  t[s]:     24.94
	FillingsUpdate:  mu: +0.513132227  nElectrons: 54.000000  magneticMoment: [ Abs: 0.00680  Tot: -0.00000 ]
	SubspaceRotationAdjust: set factor to 0.412
ElecMinimize: Iter:  13  F: -217.202948922773771  |grad|_K:  4.008e-07  alpha:  4.727e-01  linmin:  1.483e-06  t[s]:     25.88
	FillingsUpdate:  mu: +0.513130934  nElectrons: 54.000000  magneticMoment: [ Abs: 0.00568  Tot: -0.00000 ]
	SubspaceRotationAdjust: set factor to 0.462
ElecMinimize: Iter:  14  F: -217.202949072735095  |grad|_K:  2.820e-07  alpha:  5.571e-01  linmin:  8.730e-08  t[s]:     27.03
	FillingsUpdate:  mu: +0.513125601  nElectrons: 54.000000  magneticMoment: [ Abs: 0.00478  Tot: -0.00000 ]
	SubspaceRotationAdjust: set factor to 0.504
ElecMinimize: Iter:  15  F: -217.202949140086218  |grad|_K:  2.138e-07  alpha:  5.054e-01  linmin:  2.098e-07  t[s]:     28.16
	FillingsUpdate:  mu: +0.513130934  nElectrons: 54.000000  magneticMoment: [ Abs: 0.00389  Tot: +0.00000 ]
	SubspaceRotationAdjust: set factor to 0.544
ElecMinimize: Iter:  16  F: -217.202949178047390  |grad|_K:  1.624e-07  alpha:  4.954e-01  linmin:  3.686e-07  t[s]:     29.26
ElecMinimize: Converged (|Delta F|<1.000000e-07 for 2 iters).
Setting wave functions to eigenvectors of Hamiltonian
Converging empty states (this may take a while): |deigs|: 1.938e-02

Computing DFT-D3 correction:
# coordination-number Ir 6.673 6.673
# coordination-number O 2.848 2.848 2.848 2.848
# diagonal-C6 Ir 305.53 305.53
# diagonal-C6 O 10.37 10.37 10.37 10.37
EvdW_6 =   -0.021155
EvdW_8 =   -0.031255

# Ionic positions in cartesian coordinates:
ion Ir   4.264424641093000   4.264424641093000   3.003645584499000 1
ion Ir   0.000000083481000   0.000000083481000  -0.000000250174000 1
ion O   5.896306617340000   5.896306617340000  -0.000000250174000 1
ion O   1.631882059728000   6.896967222458000   3.003645584499000 1
ion O   2.632542664846000   2.632542664846000  -0.000000250174000 1
ion O   6.896967222458000   1.631882059728000   3.003645584499000 1

# Forces in Cartesian coordinates:
force Ir   0.000004781234670   0.000005264959947   0.000000825114056 1
force Ir  -0.000004395831068  -0.000005875483272  -0.000000157164922 1
force O   0.003181520838422   0.003183396098440   0.000000975641323 1
force O   0.003173219018666  -0.003173969497423  -0.000000572004379 1
force O  -0.003182185136984  -0.003182888496619   0.000001647949298 1
force O  -0.003174368517634   0.003174492441906  -0.000000754744825 1

# Energy components:
   Eewald =     -175.0227652869602935
       EH =       43.7895386148398984
     Eloc =     -136.7089284673805594
      Enl =       10.5303937757487063
     EvdW =       -0.0524097626316296
      Exc =      -64.8649067285547289
 Exc_core =       29.1043301587218401
       KE =       76.0223543716198975
-------------------------------------
     Etot =     -217.2023933245968692
       TS =        0.0005558534505231
-------------------------------------
        F =     -217.2029491780473904

IonicMinimize: Iter:   0  F: -217.202949178047390  |grad|_K:  2.119e-03  t[s]:     42.27
IonicMinimize: None of the convergence criteria satisfied after 0 iterations.

#--- Lowdin population analysis ---
# oxidation-state Ir +0.572 +0.573
# magnetic-moments Ir +0.000 -0.000
# oxidation-state O -0.259 -0.259 -0.259 -0.259
# magnetic-moments O -0.001 +0.001 -0.001 +0.001


Dumping 'jdftx.fillings' ... done
Dumping 'jdftx.wfns' ... done
Dumping 'jdftx.force' ... done
Dumping 'jdftx.n_up' ... done
Dumping 'jdftx.n_dn' ... done
Dumping 'jdftx.tau_up' ... done
Dumping 'jdftx.tau_dn' ... done
Dumping 'jdftx.d_tot' ... done
Dumping 'jdftx.eigenvals' ... done
Dumping 'jdftx.bandProjections' ... done
Dumping 'jdftx.eigStats' ... 
	eMin: -1.341990 at state 9 ( [ +0.500000 +0.500000 +0.000000 ] spin 1 )
	HOMO: +0.500298 at state 13 ( [ +0.000000 +0.000000 +0.333333 ] spin -1 )
	mu  : +0.513131
	LUMO: +0.514670 at state 21 ( [ +0.500000 +0.500000 +0.000000 ] spin -1 )
	eMax: +0.922089 at state 14 ( [ +0.000000 +0.000000 -0.333333 ] spin -1 )
	HOMO-LUMO gap: +0.014372
	Optical gap  : +0.015339 at state 14 ( [ +0.000000 +0.000000 -0.333333 ] spin -1 )
Dumping 'jdftx.Ecomponents' ... done
Dumping 'jdftx.dosUp' ... done.
Dumping 'jdftx.dosDn' ... done.
End date and time: Fri Jan  3 02:46:50 2025  (Duration: 0-0:00:43.14)
Done!

PROFILER:             augmentDensityGrid     0.000691 +/-     0.000851 s,  112 calls,      0.077422 s total
PROFILER:         augmentDensityGridGrad     0.014110 +/-     0.013752 s,   62 calls,      0.874802 s total
PROFILER:        augmentDensitySpherical     0.000831 +/-     0.000036 s, 2688 calls,      2.232573 s total
PROFILER:    augmentDensitySphericalGrad     0.000864 +/-     0.000030 s, 2996 calls,      2.589401 s total
PROFILER:                 augmentOverlap     0.000495 +/-     0.001036 s, 6280 calls,      3.107944 s total
PROFILER:                     changeGrid     0.000023 +/-     0.000003 s, 4364 calls,      0.098288 s total
PROFILER:        ColumnBundle::randomize     0.001744 +/-     0.000105 s,   24 calls,      0.041865 s total
PROFILER:                     diagouterI     0.001415 +/-     0.000080 s, 1416 calls,      2.003455 s total
PROFILER:              EdensityAndVscloc     0.001941 +/-     0.000042 s,   57 calls,      0.110627 s total
PROFILER:                     EnlAndGrad     0.000761 +/-     0.000200 s, 3188 calls,      2.426631 s total
PROFILER:            ExCorrCommunication     0.000003 +/-     0.000003 s,  355 calls,      0.001201 s total
PROFILER:               ExCorrFunctional     0.000095 +/-     0.000207 s,   61 calls,      0.005779 s total
PROFILER:                    ExCorrTotal     0.001481 +/-     0.000689 s,   61 calls,      0.090363 s total
PROFILER:                   Idag_DiagV_I     0.001656 +/-     0.001024 s, 1474 calls,      2.441028 s total
PROFILER:                    inv(matrix)     0.000308 +/-     0.001067 s,  768 calls,      0.236352 s total
PROFILER:            matrix::diagonalize     0.001007 +/-     0.003288 s, 2842 calls,      2.863020 s total
PROFILER:                    matrix::set     0.000011 +/-     0.000039 s, 24444 calls,      0.272369 s total
PROFILER:            orthoMatrix(matrix)     0.000390 +/-     0.004968 s, 1546 calls,      0.602218 s total
PROFILER:     RadialFunctionR::transform     0.005839 +/-     0.000341 s,  103 calls,      0.601433 s total
PROFILER:                    reduceKmesh     0.000022 +/-     0.000000 s,    1 calls,      0.000022 s total
PROFILER:   VanDerWaalsD3::energyAndGrad     0.652585 +/-     0.049309 s,    3 calls,      1.957755 s total
PROFILER:               WavefunctionDrag     0.064878 +/-     0.000000 s,    1 calls,      0.064878 s total
PROFILER:                            Y*M     0.000078 +/-     0.000342 s, 15987 calls,      1.253164 s total
PROFILER:                          Y1^Y2     0.000070 +/-     0.000659 s, 11480 calls,      0.801796 s total

MEMUSAGE:                   ColumnBundle     0.101409 GB
MEMUSAGE:        complexScalarFieldTilde     0.000541 GB
MEMUSAGE:                    IndexArrays     0.000668 GB
MEMUSAGE:                         matrix     0.009105 GB
MEMUSAGE:                           misc     0.001095 GB
MEMUSAGE:                    ScalarField     0.009012 GB
MEMUSAGE:               ScalarFieldTilde     0.003219 GB
MEMUSAGE:                          Total     0.116929 GB
